client-session/src/main/resources/archetype-resources/src/test/java/AppTest.java

```diff
@@ -0,0 +1,38 @@
+package $com.iluwatar;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
+    extends TestCase
```

Comment:
I think you are using an old version of JUnit here. Please upgrade to the latest version.

Reply:
Fixed.